### PR TITLE
update to v18; operator file for persistent_homes

### PR DESCRIPTION
**Pull request by drnic**
```
jumpbox/0:~$ pwd
/var/vcap/store/home/jumpbox
```

### CODE REVIEW

File: operators/persistent-homes.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- type: replace
+  path: /instance_groups/name=jumpbox/jobs/name=user_add/properties/persistent_homes?
+  value: true
+
+- type: replace
+  path: /instance_groups/name=jumpbox/persistent_disk_pool?
```
Comment (cppforlife) [inline operators/persistent-homes.yml:6]: needs to be under properties.

Comment (drnic) [reply operators/persistent-homes.yml:6]: argh, sorry about that. Fixed.

**Comment by drnic** [comment]
Bumped to v18 of os-conf & fixed issue with missing `properties?`

File: operators/persistent-homes.yml (hunk 1 above)
Comment (drnic) [reply operators/persistent-homes.yml:6]: Actually, not sure what you mean. `persistent_disk_pool` isn't part of properties.

Comment (drnic) [reply operators/persistent-homes.yml:6]: But I was missing `disk_pools` in the base jumpbox.yml

**Comment by drnic** [comment]
I'll close this PR. You'll bump the os-conf version when you're ready. I'm patching it in my use cases.

**Pull request closed without merging**